### PR TITLE
[11.0][MIG] l10n_ch_hr_payroll

### PR DESCRIPTION
**Pull request by antonioburic**
Auto migrate python 2 to 3
Remove coding from headers
Updated readme directory
Also some translated terms to DE added
by @lucode

### CODE REVIEW

File: l10n_ch_hr_payroll/models/hr_payslip.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            elif payslip.working_days == 0 and payslip.non_working_days == 0:
+                payslip.working_rate = 100
+
+    @api.multi
+    def compute_sheet(self):
+        for payslip in self:
+            payslip._onchange_working_non_working_days()
+            payslip._onchange_employee_worked_hours()
+            res = super(HrPayslip, payslip).compute_sheet()
+        return res
```
Comment (yvaucher) [inline l10n_ch_hr_payroll/models/hr_payslip.py:73]: What if self is composed of a len(recordset) > 1. `res` will only contains the return value of the last record.
I think you need to aggregate all returned values.

File: l10n_ch_hr_payroll/models/hr_salary_rule.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                            ('id', '=', data_id.res_id)
+                        ])
+
+                        if rule_to_modify.id == rule.id:
+                            rule.percentage = \
+                                getattr(rule.company_id, rule_from)
+
+    @api.multi
+    def _compute_rule(self, localdict):
+        res = super(HrSalaryRule, self)._compute_rule(localdict)
```
Comment (yvaucher) [inline l10n_ch_hr_payroll/models/hr_salary_rule.py]:
```suggestion
        res = super()._compute_rule(localdict)
```

Let's make it python3

**Review by yvaucher — CHANGES_REQUESTED** [review]
Thanks for the migration.

Have you run `2to3`?

If not please follow the steps https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: l10n_ch_hr_payroll/models/hr_payslip.py (hunk 1 above)
Comment (antonioburic) [reply l10n_ch_hr_payroll/models/hr_payslip.py:73]: thanks, edited!

File: l10n_ch_hr_payroll/models/hr_salary_rule.py (hunk 2 above)
Comment (antonioburic) [reply l10n_ch_hr_payroll/models/hr_salary_rule.py]: changed ^^ 
2to3 applied.

**Comment by yvaucher** [comment]
@antonioburic I don't see your last changes. You might want to do a `git push` :)

**Comment by antonioburic** [comment]
@yvaucher ah yes sorry, my bad --  you're right, I rushed and pushed to the wrong branch initially (11.0-mig-l10n_ch_hr_payroll_report), moved now ^^

**Pull request merged**